nearcore/src/runtime/mod.rs

```diff
@@ -517,6 +525,21 @@ impl NightshadeRuntime {
                epoch_height, &epoch_id, current_protocol_version, is_first_block_of_version,
         );
 
+        #[cfg(feature = "snapshot_grabber")]
```

Comment:
I'm surprised that we have to modify runtime code for this tool - I'd rather have this tool 'separate'